akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ConsumerControllerImpl.scala

```diff
@@ -414,6 +432,10 @@ private class ConsumerControllerImpl[A](
       }
   }
 
+  private def clearStashBuffer(): Unit = {
+    stashBuffer.unstashAll(Behaviors.ignore)
```

Comment:
Is there a `clear` missing on `StashBuffer`?

Reply:
yup, added issue https://github.com/akka/akka/issues/28786